types/elliemae__em-ssf-guest/elliemae__em-ssf-guest-tests.ts

```diff
@@ -0,0 +1,28 @@
+import emssf from "elliemae__em-ssf-guest";
```

Comment:
You should still use the pathed name - this will check that the tsconfig is properly configured
```suggestion
import emssf from "@elliemae/em-ssf-guest";
```

Reply:
Ah makes sense. Thank you for reviewing, this has been fixed.